pkg/kudoctl/util/repo/repo_framework.go

```diff
@@ -83,7 +76,7 @@ func (r *FrameworkRepository) GetPackage(bundleName string) (*PackageCRDs, error
 	}
 
 	// first try old package format
```

Comment:
what is meant here with "first", I would expect a second following first...

And then "old package format" (I know this is also not really part of this pr) but help me understand:
It always extracts the package to the "old" (not new kep-9 conform) version and translates if it is a "new" (kep-9 conform) version back into the old one. and the "old" one is the struct `PackageCRDs` and the new one is the struct `v1Package` ?

Reply:
Ok, I renamed a few things. 

1. `v1Package -> PackageFiles` since that is basically what it is.
2. `readV1PackageFile -> parsePackageFile` since it also unmarshals it
3. `extractV1Package -> parsePackage` since it also parses files and return `*PackageFiles` now